### PR TITLE
update p4-fusion doc

### PR DESCRIPTION
**Pull request by sashaostrikov**
Follow-up to https://github.com/sourcegraph/sourcegraph/pull/41113

### CODE REVIEW

File: content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Successfully tagged p4-fusion:latest
 Run the binary in the container by running the following command:
 
 ```
-docker run -t -i p4-fusion /sr/local/bin/p4-fusion
+docker run -t -i p4-fusion /usr/local/bin/p4-fusion
```
Comment (indradhanush) [inline content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md:77]: Good catch! 👌

File: content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ You'll need to configure docker so that the images build for amd64, not arm64.
 
 ```sh
 docker buildx create --use
-docker buildx build -t p4-fusion --platform linux/amd64 --target=p4-fusion .
+docker buildx build --load -t p4-fusion --platform linux/amd64 --target=p4-fusion .
```
Comment (indradhanush) [inline content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md:41]: Interesting. Is this flag something that was always needed but didn't have it in the docs? Cannot recall needing this flag the last time I upgraded p4-fusion.

**Review by indradhanush — APPROVED** [review]

File: content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md (hunk 2 above)
Comment (indradhanush) [reply content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md:41]: Oh I think I recall now. I had some difficulties with building this on OSX and booted up my Linux work machine to do it instead which might explain why I didn't need the flag there. Do you want to clarify that this is for OSX?

Comment (indradhanush) [reply content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md:41]: Haha! Never mind. This **is** a section for OSX.

Comment (sashaostrikov) [reply content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md:41]: 😀😀😀

**Pull request merged**